### DISCUSSION TITLE
Embed files

### DISCUSSION THREAD

**Discussion by DxFeelgood** [Quantum-Manager/tracker#74] [Q&A]
Hi! I'm new with Quantum Manager, I'm building a website with Joomla! 3.9.25, PHP version 7.3.27.
I'm stuck in the embed files window, I click on the "Embed" button after selecting an image or a PDF document, but nothing happens.
I have no errors in the chrome console and when I click on the button I see nothing going in the Network tab.
Could you please help me?

![Embed](https://user-images.githubusercontent.com/81106865/111916827-1ef74a00-8a7d-11eb-81ee-db0fda77f8e2.png)

**Comment by dmitriitux** [comment]
Hi, yes, I know about the problem, I will release the update within a few hours

С уважением, Цымбал Дмитрий +7 991 111 56 42 (Telegram, WhatsApp) https://delo-design.ru
21 марта 2021 г., 21:40 +0300, DxFeelgood ***@***.***>, писал:

**Reply by DxFeelgood** [reply]
Thank you very much for the reply!

**Reply by dmitriitux** [reply]
released a fix version

**Reply by DxFeelgood** [reply]
It works! Thank you very much! Superfast!
